website/docs/components/table/partials/content/badge-usage.md

```diff
@@ -0,0 +1,70 @@
+## Badge usage
+
+Use [Badges](/components/badge) to communicate status and high-priority metadata within a Table.
+
+### Badge type
+
+We recommend using `outlined` [Badges](/components/badge) within a Table. This variant provides enough differentation between the component, the value in the cell, and the background of the Table row.
+
+![Outlined Badges within a Table](/assets/components/table/outlined-badge-within-Table.png)
```

Comment:
This image appears to be broken

Reply:
Shoot, thanks for the note on that. I did a bulk text replace to fix some capitalization issues and it replaced the file paths as well. I'll get em cleaned up!